### PR TITLE
Pull toastr from npm instead of bower

### LINKED ISSUE #14: Remove bower dependency
Hi guys

Could you work towards removing the Bower dependency and use yarn/npm instead please as ember itself is doing.

Thanks
Caltor

### PR DESCRIPTION
**Pull request by zomarg**
Resolve #14 
Other changes:
Travis build file changed to test for node 4 and 6.
Remove testing for older nodes beacuse of ES6 language support.

Tests passing locally, travis is green also.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,17 +1,50 @@
...
       target = target.app;
     }
-    var bowerDir = target.bowerDirectory;
 
-    target.import(bowerDir + '/toastr/toastr.js');
-    target.import(bowerDir + '/toastr/toastr.css');
+    target.import(
+        {
+          development: vendor + '/toastr/toastr.js',
+          production: vendor + '/toastr/build/toastr.min.js'
```
Comment (knownasilya) [inline index.js]: This shouldn't be necessary since ember-cli does a production build itself.

File: index.js [hunk 2]
```diff
@@ -1,17 +1,50 @@
...
+        }
+    );
+    target.import(
+        {
+          development: vendor + '/toastr/build/toastr.css',
+          production: vendor + '/toastr/build/toastr.min.css'
```
Comment (knownasilya) [inline index.js]: Same as above

File: package.json [hunk 3]
```diff
@@ -43,7 +45,8 @@
     "toastr"
   ],
   "dependencies": {
-    "ember-cli-babel": "^5.0.0"
+    "ember-cli-babel": "^5.0.0",
+    "toastr": "^2.1.2"
```
Comment (knownasilya) [inline package.json]: If this gets added to the user's project, it shouldn't be here (only in dev deps for dummy app).

**Review by knownasilya — CHANGES_REQUESTED** [review]
Thanks for this! Just a few comments, but everything else looks good.

File: package.json (hunk 3 above)
Comment (zomarg) [reply package.json]: Great thanks for the comments will do the changes :D

Comment (knownasilya) [reply package.json]: Just for toastr, not for ember-cli-babel. That needs to stay for transpilation of the addon.

**Comment by roberkules** [comment]
i just tried to install `ember-toastr` and I noticed that it's still showing the `Install toastr.js from bower` blueprint. I guess it's because you didn't bump the package version (to `1.5.1` or `1.6`)?!

**Comment by knownasilya** [comment]
I just merged this and haven't published the changes yet. Will try to get to it later today.

**Comment by roberkules** [comment]
alright, sorry for being impatient ;)

**Comment by zomarg** [comment]
@knownasilya is there any way you could publish to npm please

**Comment by knownasilya** [comment]
All set @GramozKrasniqi

**Pull request merged**